content/docs/nav.yml

```diff
@@ -74,6 +76,8 @@
       title: Accessibility
     - id: code-splitting
       title: Code-Splitting
+    - id: strict-mode
+      title: StrictMode
```

Comment:
Maybe “Strict Mode” as a title? It seems like this section uses concept names rather than API names for titles. Eg we have “forwarding refs” not “forwardRef”.